asv_bench/asv.conf.json

```diff
@@ -53,6 +53,10 @@
     // the base URL to show a commit for the project.
     "show_commit_url": "https://github.com/modin-project/modin/commit/",
 
+    // The Pythons you'd like to test against.  If not provided, defaults
+    // to the current version of Python used to run `asv`.
+    "pythons": ["3.8"],
```

Comment:
Python 3.8 was used before, but it's better to set the version explicitly, as in other configurations.

Reply:
Do we need the code duplication below?

Reply:
Removed